packages/client/tests/functional/tracing/tests.ts

```diff
@@ -554,9 +554,9 @@ testMatrix.setupTestSuite(
   },
   {
     skipProviderFlavor: {
-      from: ['js_libsql'],
+      from: ['js_libsql', 'js_planetscale'],
       reason:
-        'The spans are not consistent, or not in a way that makes sense compared to the order driver adapters. Needs investigation.',
+        'The spans are not consistent, or not in a way that makes sense compared to the other driver adapters. Needs investigation.',
```

Comment:
This is what I said before: libSQL and PlanetScale adapters use high-level APIs for transactions provided by the underlying drivers and we don't run `BEGIN` and `COMMIT`/`ROLLBACK` statements ourselves, hence no spans for them.

Reply:
If there are any other issues, those would need investigation, but if it's just missing spans for `BEGIN`/`COMMIT`, then it's well understood and expected (what we want to or even could do about it though is another question).